### PR TITLE
Book model created

### PR DESCRIPTION
**Pull request by santyalcala**
# Summary

- Model Book created, with a migration for not null columns.

# Trello card

- https://trello.com/c/ay5PFvl0/21-crear-modelo-de-book

# Screenshot

-
books

### CODE REVIEW

File: app/models/book.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class Book < ApplicationRecord
```
Comment (Dinuuu) [inline app/models/book.rb]: I think you forgot to add some validations to the model

File: db/migrate/20170627205216_make_not_null_books_fields.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class MakeNotNullBooksFields < ActiveRecord::Migration[5.0]
+  def change
+    change_column_null :books, :genre, false
```
Comment (Dinuuu) [inline db/migrate/20170627205216_make_not_null_books_fields.rb]: remember that you can set this in the same migration you create the table (leave it like this now, its for the next model)

Comment (santyalcala) [inline db/migrate/20170627205216_make_not_null_books_fields.rb]: I wanted to do that but I couldn't, I read that when I create the model with the generator I can't use any parameter like not null, etc. Just the columns and types.

File: app/models/book.rb (hunk 1 above)
Comment (santyalcala) [inline app/models/book.rb]: Yes, I added them in the new branch of tests, but should I add them here and then rebase?

Comment (santyalcala) [inline app/models/book.rb]: Added to books and then did the rebase from books-tests.

**Pull request merged**